src/_pytest/pytester.py

```diff
@@ -634,46 +661,48 @@ def __init__(self, request: FixtureRequest, tmpdir_factory: TempdirFactory) -> N
         else:
             name = request.node.name
         self._name = name
-        self.tmpdir = tmpdir_factory.mktemp(name, numbered=True)
-        self.test_tmproot = tmpdir_factory.mktemp("tmp-" + name, numbered=True)
+        self._path: Path = tmp_path_factory.mktemp(name, numbered=True)
```

Comment:
Why the tmproot removal?

Reply:
Hmmm it seems it was not really used, but I will reintroduce it in `Testdir`, good catch. 👍